Solutions/GitHub/Hunting Queries/User First Time Repository Delete Activity.yaml

```diff
@@ -0,0 +1,26 @@
+id: c3237d88-fdc4-4dee-8b90-118ded2c507c
+name: GitHub First Time Repo Delete
+description: |
+  'This hunting query identifies GitHub activites its the first time a user deleted a repo that may be a sign of compromise.'
+requiredDataConnectors: []
```

Comment:
Please add the connectorid and datatype in this format
requiredDataConnectors: -
connectorId: connector id
dataTypes: - name of the parser which we use in the query

Thanks,
Aditi Maheshwari

Reply:
Added